staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -182,19 +172,14 @@ func getRuntimeConfigValue(overrides utilflag.ConfigurationMap, apiKey string, d
 func ParseGroups(resourceConfig utilflag.ConfigurationMap) ([]string, error) {
 	groups := []string{}
 	for key := range resourceConfig {
-		if key == "api/all" || key == "v1" {
+		if key == "api/all" {
```

Comment:
will this break the `v1` case? Does it still parse correctly below?

Reply:
I pre processed it just delete it. Because in original code , I tested the `v1` flag does not work.

Reply:
:+1: